js/widgets/forms/clearButton.js

```diff
@@ -13,6 +13,11 @@ define( [
 
 	$.widget( "mobile.textinput", $.mobile.textinput, {
 		options: {
+			classes: {
+				"ui-textinput-clear-button":
+					"ui-button ui-button-icon-only ui-corner-all ui-button-right",
+				"ui-textinput-clear-button-icon": "ui-icon-delete ui-icon"
```

Comment:
Icons are not generally part of classes, though there is some debate about this still, icons are generally their own option.